crates/rolldown/tests/rollup/relative-external/artifacts.snap

```diff
@@ -0,0 +1,35 @@
+---
+source: crates/rolldown_testing/src/integration_test.rs
+---
+# Assets
+
+## main.js
+
+```js
+import "./external";
+
+```
+---
+
+Variant: ()
+
+# Assets
+
+## main.js
+
+```js
+import "./external";
+
+```
+---
+
+Variant: ()
+
+# Assets
+
+## chunk/main.js
+
+```js
+import "../external";
```

Comment:
The test you could see the rendered `./external` will changed if set `entryFilename: 'chunk/[name].js'`,